app/controllers/webhooks/mailchimp_unsubscribes_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Webhooks::MailchimpUnsubscribesController < ApplicationController
+  class InvalidListID < StandardError; end
+
+  LIST_MAPPINGS = {
```

Comment:
Is there a better way to do this lookup/is this mapping defined somewhere already?

Reply:
This is the first of it's kind 😄

Reply:
Ok, good to know. Can you please sanity check the mapping?